server/menus/tests.py

```diff
@@ -11,22 +13,74 @@ def setUp(self):
         self.test_user = User.objects.create(username="Foxy",
                                              first_name="Foxy")
         self.chef = Chef.objects.create(user=self.test_user)
-
+        self.menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
+        self.item = Item.objects.create(menu=self.menu, title="kibbles", size="the whole bowl",
+                                   price=13.33, description="food for foxy")
     def test_menu_creation(self):
         '''
         Makes sure menu is properly created and linked to chef.
         '''
 
-        # Tests creation of menu and connection to user
-        self.assertFalse(Menu.objects.filter(chef=self.chef).exists())
-        menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
-        self.assertTrue(Menu.objects.filter(chef=self.chef).exists())
-        self.assertEqual(self.chef, menu.chef)
-        self.assertEqual(True, menu.delivery)
+        # # Tests creation of menu and connection to user
+        # self.assertFalse(Menu.objects.filter(chef=self.chef).exists())
+        # menu = Menu.objects.create(chef=self.chef, title='foxy food', delivery=True )
+        # self.assertTrue(Menu.objects.filter(chef=self.chef).exists())
+        # self.assertEqual(self.chef, menu.chef)
+        # self.assertEqual(True, menu.delivery)
+
+        # # Tests creation of item and connection to menu
+        # self.assertFalse(Item.objects.filter(menu=menu).exists())
+        # item = Item.objects.create(menu=menu, title="kibbles", size="the whole bowl",
+        #                            price=13.33, description="food for foxy")
+        # self.assertTrue(Item.objects.filter(menu=menu).exists())
+        # self.assertEqual(menu, item.menu)
+
+    def test_menu_get(self):
+        '''
+        Tests GET metho for menu-detail and item-detail
+        '''
+
+        url = reverse('menu-detail', kwargs={'pk':self.menu.id})
+        response = self.client.get(url)
+        self.assertEqual(response.data, {'id': 1, 'chef':1, 'title':'foxy food', 'delivery':True})
```

Comment:
this is a general note to change in a few of your tests - it's just something good to get used to cause most testers do it this way. anyway in your assertions it should be "expected, actual". So inside the assertEqual first you put the values you expect (the dictionary with all the data you are hoping is returned in response.data) and THEN you put the ACTUAL values you get back (meaning the response.data object). Make sense? We should prob change it to that in a few places.